web/app/js/components/ResourceList.jsx

```diff
@@ -51,7 +51,7 @@ export class ResourceListBase extends React.Component {
       <MetricsTable
         resource={friendlyTitle}
         metrics={processedMetrics}
-        linkifyNsColumn={true} />
+        showNsColumn={true} />
```

Comment:
This feels like something we should add to the tests, just to make it explicit that if you're doing a `ResourceList`, you're showing the ns column.

Reply:
Good suggestion, tests added. Plus I added prop type validation for this component and cleaned up some unused props / params.